index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <div class="container">
+        <header>
+            <div class="right-side">
+                <h3>Anicet Fantômas</h3>
+            </div>
```

Comment:
- [ ] Logo is supposed to be a link because users generally click on logo to go to home page. Let's wrap logo with `<a>` tag
- [ ] Please add more space under navbar as provided in figma